src/Home.py

```diff
@@ -56,26 +58,77 @@ def get_df():
 st.markdown("Our team is working to understand and inform others about deaths of migrants, focused at borders. We are relying on the [Missing Migrants Project](https://missingmigrants.iom.int/) dataset, which includes data spanning from 2014 until 2022. This data includes incidents across the world, each dot on the map involving the death of 1 or more persons, with details about the location, cause of death, time, and more if it can be identified.  In this project, we use this data to create an interactive data visualization map of the world, allowing readers to see how migrant deaths vary by location, cause of death, gender, and season. Additionally, we enrich the project with more recent data from Twitter by filtering and pulling down Tweets about migrants.")
 st.markdown("This migration data is more than just numbers. This is information about real people that left their home countries for various reasons and ultimately perished during their journey. This is data that requires respect and care in order to be presented faithfully. In this project, we aim to bring awareness to the challenges and risks that migrants face everyday. Perhaps you are familiar with several of the migration routes displayed here. There are likely other dangerous routes that you maybe haven't heard of. Many of us can recall certain newstories that made it to the news about the deaths of migrants, but as we can see in the data, this is an ongoing, everyday risk that affects many individuals, families, communitites, and countries around the world.")
 st.markdown("We welcome you to explore the below visualization by zooming into the map directly or by using the filters in the left side menu.")
+
+#map_w = folium.Map()
+#loca = (migrantdf.iloc[0]['lon'],migrantdf.iloc[0]['lat'])
+#custom_marker = folium.Marker(location=loca)
+#custom_marker.add_to(map_w)
+#st_map = st_folium(map_w)
 # 1.2 map
-st.map(migrantdf)
+#st.map(migrantdf)
+
+fig = go.Figure(data=go.Scattergeo(
+        lon = migrantdf['lon'],
+        lat = migrantdf['lat'],
+        text = migrantdf['Cause of Death Abbreviation'],
+        #name=migrantdf['Migration route'],
+        hoverinfo=['name','text',"lon+lat"],
+        mode = 'markers',
+        marker_color = 'rgb(210,30,0)',
+        marker=dict(
+            colorscale='Reds'
+        )
+        ))
+
+fig.update_layout(
+    title = 'Locations of the Reports',
+    autosize=False,
+    #width=750,
+    #height=400,
+    geo = dict(
+        scope='world',
+        showcountries = True,
+        showocean = True,
+        landcolor = "#f2f2f0",
+        oceancolor = "#cad2d3",
+        showcoastlines = False,
+    ),
+    margin={"r":0,"t":0,"l":0,"b":0},
+    )
+    
+
+st.plotly_chart(fig)
```

Comment:
Wow this is awesome. 
<img width="731" alt="Screen Shot 2022-10-19 at 09 31 45" src="https://user-images.githubusercontent.com/53492091/196614174-ad71e344-e970-4fdd-b7dc-3681e27a605c.png">

Reply:
I wonder if we should also include the number of those killed per dot?